### PR TITLE
change default backend for lightning

### PR DESCRIPTION
**Pull request by KanShiori**
### Code changes

- [ ] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [ ] Manual test 
- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note
change default backend of lightning from impoter to local
```

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]
LGTM. add a release note?

**Comment by KanShiori** [comment]
> LGTM. add a release note?

Done

File: charts/tidb-lightning/values.yaml [hunk 1]
```diff
@@ -91,9 +91,9 @@ annotations: {}
...
 affinity: {}
 
-# The delivery backend used to import data (valid options include `importer`, `local` and `tidb`).
+# The delivery backend used to import data (valid options include `local`, `tidb` and `importer`(deprecated)).
 # If set to `local`, then the following `sortedKV` should be set.
-backend: importer
+backend: local
 
 # For `local` backend, an extra PV is needed for local KV sorting.
 sortedKV: {}
```
Comment (DanielZhangQD) [inline charts/tidb-lightning/values.yaml:99]: The `storageClassName` and `storage` should be uncommented.

Comment (KanShiori) [reply charts/tidb-lightning/values.yaml:99]: Done

File: charts/tidb-lightning/values.yaml [hunk 2]
```diff
@@ -91,14 +91,14 @@ annotations: {}
...
-#  storageClassName: local-storage
-#  storage: 100Gi
+storageClassName: local-storage
+storage: 100Gi
```
Comment (DanielZhangQD) [inline charts/tidb-lightning/values.yaml:101]:
```suggestion
  storageClassName: local-storage
  storage: 100Gi
```

File: charts/tidb-lightning/values.yaml [hunk 3]
```diff
@@ -91,14 +91,14 @@ annotations: {}
...
 affinity: {}
 
-# The delivery backend used to import data (valid options include `importer`, `local` and `tidb`).
+# The delivery backend used to import data (valid options include `local`, `tidb` and `importer`(deprecated)).
 # If set to `local`, then the following `sortedKV` should be set.
-backend: importer
+backend: local
 
 # For `local` backend, an extra PV is needed for local KV sorting.
 sortedKV: {}
```
Comment (DanielZhangQD) [inline charts/tidb-lightning/values.yaml:99]:
```suggestion
sortedKV:
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**